google-oauth-client-java6/src/main/java/com/google/api/client/extensions/java6/auth/oauth2/AuthorizationCodeInstalledApp.java

```diff
@@ -37,9 +37,39 @@
  *
  * @since 1.11
  * @author Yaniv Inbar
+ * @author Philipp Hanslovsky
  */
 public class AuthorizationCodeInstalledApp {
 
+
+  /**
+   *
+   * Helper interface to allow caller to browse.
+   *
+   */
+  public static interface Browser {
+    /**
+     *
+     * @param url url to browse
+     * @throws Exception
+     */
+    public void browse(String url) throws Exception;
```

Comment:
Is there a need for this to be `Exception` rather than `IOException`?

Reply:
No particular reason from my side, I will update the interface.